src/main/java/duke/app/Duke.java

```diff
@@ -0,0 +1,62 @@
+package duke.app;
+
+import duke.command.Command;
+import duke.exception.DukeException;
+import duke.parser.Parser;
+import duke.storage.Storage;
+import duke.tasklist.Tasklist;
+import duke.ui.Ui;
+
+import java.util.NoSuchElementException;
```

Comment:
I like how you separate the class packages and the java packages. It's neater, especially when there is more imports. 👍